library/Fission/Web/Auth.hs

```diff
@@ -54,3 +71,33 @@ checkUser (BasicAuthData username password) = do
          else do
            logWarn <| "Unauthorized user! Username: " <> displayBytesUtf8 username
            return Unauthorized
+
+jwt :: 
+  ( MonadIO (RIO cfg)
+  , MonadSelda (RIO cfg)
+  , HasLogFunc cfg
+  )
+  => RIO cfg (AuthHandler Request User)
+jwt = 
+  ask >>= \cfg -> 
+    pure <| mkAuthHandler <| toHandler cfg . jwtHandler
+
+jwtHandler ::
+  ( MonadIO (RIO cfg)
+  , MonadSelda (RIO cfg)
+  , MonadThrow (RIO cfg)
+  )
+  =>  HasLogFunc cfg
+  => Request
+  -> RIO cfg User
+jwtHandler req =
+  JWT.validateReq req >>= \case
+    Left err -> throwM err
+    Right did -> do
+      logDebug <| "DID: " <> displayShow did
+      -- @@TODO: this part just filler until we update DB:
```

Comment:
As the comment says, we'll want to actually look up the user w/ their DID. But we need DB stuff settled before doing this